src/tools/create-callback-wrapper.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const _ = require('lodash');
+
+//this returns a higher order function that allows users to generate the callback URL
+//generation doesn't actually generate the URL (although we could use RPC if we wanted to)
+//instead it relies on the callbackUrl being passed in by the platform
+//as such it'll always return the same value.
+//calling this inner function will also mark the task as a CALLBACK status on return which will
+//effectively pause the Zap
+
+const createCallbackHigherOrderFunction = input => {
+  let callbackUrl = _.get(input, '_zapier.event.callbackUrl');
+  return function() {
```

Comment:
minor style thing, we mostly use fat arrow syntax, so I'd change to `return () => {`